frontends/ol-ckeditor/src/components/CkeditorDisplay.tsx

```diff
@@ -68,12 +68,15 @@ const CkeditorDisplay: React.FC<{
     [dangerouslySetInnerHTML],
   )
   return (
-    <div
-      className={classNames("ck-content", className)}
-      dangerouslySetInnerHTML={{
-        __html: html,
-      }}
-    />
+    <>
+      <Styles />
```

Comment:
The
```
  .ck.ck-balloon-panel.ck-powered-by-balloon,
  .ck.ck-balloon-panel_visible.ck-powered-by-balloon {
    display: none !important;
  }
```
styles should be loaded with the editors, not with the read-only display.

For loading the above CSS (which is currently in Styles.tsx), I think either
- a component that wraps CkeditorArticle and CkeditorMarkdown 
- or a `.css` file like the vendor one
    - honestly I think this would be OK. We don't need anything dynamic, it's already scoped to only affect ckeditor stuff, and that way it's loaded exactly once, as opposed to once per editor. (Although, probalby would only have one editor on screen at a time.)